symengine/lib/symengine_wrapper.pyx

```diff
@@ -16,6 +16,10 @@ import warnings
 
 include "config.pxi"
 
+from setup import use_cysignals
+if use_cysignals:
+    include "cysignals/signals.pxi"
```

Comment:
You need to use compile time constants like what we do with constant like `HAVE_SYMENGINE_MPFR`